lib/modules/ens/embarkjs.js

```diff
@@ -0,0 +1,297 @@
+const namehash = require('eth-ens-namehash');
+/*global web3*/
+let __embarkENS = {};
+
+// registry interface for later
+var registryInterface = [
```

Comment:
put this one and the vars inside ```__embarkENS``` so avoid any collisions in the global namespace